### DISCUSSION TITLE
How to add random crop augmentation in yolov5 segmentation training?

### DISCUSSION THREAD

**Discussion by Siddhantjad** [ultralytics/yolov5#11014] [Q&A]
Hi is there a way to add random crop augmentation in yolov5 segmentation training pipeline?,  any suggestions will be helpful.

**Comment by dudas-tamas** [comment]
I hope it will help you. There is an albumentations integration in yolov5, which means you can augment yout images while training. You can search for more albumentation component to use in this repo: https://github.com/albumentations-team/albumentations/tree/master/albumentations/augmentations. For more info: https://github.com/albumentations-team/albumentations/issues/949
